src/components/common/header/UnauthenticatedHeader.js

```diff
@@ -17,12 +17,14 @@ export default () => (
           <li><Link to="*">SCI-FI</Link></li>
           <li><Link to="*">CULTURE</Link></li>
           <li><Link to="*">MORE...</Link></li>
-          <li><a className="waves-effect waves-light btn" href="#!">login</a></li>
+          <li><Link className="waves-effect waves-light btn" to="/signin">sign in</Link></li>
```

Comment:
I don't remember what the mockup has but shouldn't we have an option to signup on the header?

Reply:
Yes, the mockup doesn't have one, but it would make sense to include it.